crates/evm/evm/src/executors/mod.rs

```diff
@@ -822,7 +834,12 @@ impl Default for RawCallResult {
             coverage: None,
             transactions: None,
             state_changeset: HashMap::default(),
-            env: EnvWithHandlerCfg::new_with_spec_id(Box::default(), SpecId::LATEST),
+            env: Env::new_with_spec_id(
```

Comment:
is this the same as `env: Env::default(),`?

Reply:
Nice, fixed in https://github.com/foundry-rs/foundry/pull/10183/commits/5ad9091c21aa7c5d2e74513dd0fbf16f4d64cbcd